rest-api/pom.xml

```diff
@@ -106,21 +106,21 @@
 		<dependency>
 			<groupId>net.jodah</groupId>
 			<artifactId>concurrentunit</artifactId>
-			<version>0.4.2</version>
+			<version>0.4.3</version>
 			<scope>test</scope>
 		</dependency>
 
 		<dependency>
 			<groupId>cloud.localstack</groupId>
 			<artifactId>localstack-utils</artifactId>
-			<version>0.1.4</version>
+			<version>0.1.7</version>
 			<scope>test</scope>
 		</dependency>
 
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-test</artifactId>
-			<version>4.3.10.RELEASE</version>
+			<version>5.0.1.RELEASE</version>
```

Comment:
This is a big jump here.  Plus, SpringBoot version 1.5.8 is still on the Spring 4.* release.  I would keep this at the latest and greatest Spring 4.* release.

Reply:
It's just the testing framework, and all tests pass.

Reply:
It doesn't try to bring in the Spring 5.0 libraries with this?

Reply:
There's an <exclusions> block which excludes *.